### DISCUSSION TITLE
Welcome to appleMusic-to-Spotify Discussions!

### DISCUSSION THREAD

**Discussion by WestheadJ** [WestheadJ/AppleMusic-to-Spotify-Converter-Python#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MeganFang** [comment]
Apple Music provides various kinds of songs for me to enjoy. I'm really stuck on it. But when I want to download songs, it's pretty difficult because of the DRM encryption. Thankfully, I could do this via some Apple Music Converters. The one I'm currently using is of DumpMedia. It has fast conversion speed and What do you guys think about it? Do you have any other recommendations?
